### PR TITLE
Fixed multiple tables error in simd_const.wast

### PR DESCRIPTION
**Pull request by Honry**
Fixed @binji's comment at https://github.com/WebAssembly/simd/pull/132#commitcomment-37757400

### CODE REVIEW

**Comment by Honry** [comment]
@binji, PTAL. Thanks!

File: test/core/simd/simd_const.wast [hunk 1]
```diff
@@ -951,11 +944,17 @@
...
+  (func (export "as-call_indirect-param") (result v128)
+    (call_indirect (type $sig)
+      (v128.const i32x4 0 1 2 3) (v128.const i32x4 0 1 2 3) (v128.const i32x4 0 1 2 3) (i32.const 0)
+    )
+  )
   (func (export "as-call_indirect-param2") (result v128)
-    (call_indirect (type $sig2)
-      (v128.const i64x2 0 1) (v128.const i64x2 0 1) (v128.const i64x2 0 1) (i32.const 0)
+    (call_indirect (type $sig)
+      (v128.const i64x2 0 1) (v128.const i64x2 0 1) (v128.const i64x2 0 1) (i32.const 1)
```
Comment (binji) [inline test/core/simd/simd_const.wast]: Ah, I forgot about changing the index too. Thanks for checking!

**Comment by arunetm** [comment]
LGTM

**Comment by Honry** [comment]
@arunetm, could you help merge it? Since it's been approved but still open for a while.

**Comment by arunetm** [comment]
Thanks @Honry

**Pull request merged**